zone/client_packet.cpp

```diff
@@ -14460,21 +14460,7 @@ void Client::Handle_OP_TargetCommand(const EQApplicationPacket *app)
 		}
 		else
 		{
-			SetTarget(nullptr);
```

Comment:
This seems innocent enough, do you think we'll have unintended side effects for removing all of these?

Reply:
Sounds good. Let's send it